.github/workflows/snyk-scan.yml

```
@@ -17,11 +21,12 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run Snyk to check for vulnerabilities
-        uses: snyk/actions/golang@master
+        uses: snyk/actions/golang@cdb760004ba9ea4d525f2e043745dfe85bb9077e
         continue-on-error: true # To make sure that SARIF upload gets called
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
+          snyk-version: v1.1297.3
```

```suggestion
          snyk-version: latest
```
as far as I know it is recommended always to have the latest cli